tests/test_ssrf_filter.py

```diff
@@ -311,8 +362,8 @@ def test_tls_with_SNIs_supported(tmp_path):
         # Ensure when SNI support is disabled client-side, then
         # it no longer works, which allows to be sure the test is indeed testing SNI.
         with pytest.raises(SSLError) as exc_info:
-            with disable_sni_support():
-                assert do_request().status_code == 200
+            http_manager.config.ip_filter_tls_sni_support = False
+            assert do_request().status_code == 200
```

Comment:
Shouldn't this line just say `do_request()`?

Reply:
Good catch! Will fix it in a future commit!

Reply:
Pushed a fix at e4c6f10c6c3b3ca2c51cfcf51ed459932b750241, thanks!